docs/integration/local-development.md

```diff
@@ -46,3 +46,20 @@ If you encounter this error, ensure that x-dash is set up correctly in the paren
 ## Avoid linking
 
 Usually, using a locally-installed version of a package is a use case for `npm link`. In practice, we have found it to be brittle, causing problems with peer dependencies and nested transitive dependencies. Using relative `npm install` treats the installed package as any other, ensuring that your `node_modules` directory has the expected structure.
+
+## Build one component only
+
+If you don't want to build all the components while you are working on one, there is a convenience `make` command that allows you to limit builds by keyword
+
+* Add a keyword to your `package.json` file
+
+```javascript
+keywords: [
+	'x-name-of-component'
+]
+```
```

Comment:
By default `--filter` works on the name property, to target keywords you'd have to do something like this:

```sh
athloi run build --filter 'keywords:keyword'
```
```

Reply:
ah ok, I've been using x-audio but then the keywords and the name are identical. will update the docs to use name